utility_network/perform-valve-isolation-trace/README.md

```diff
@@ -0,0 +1,53 @@
+# Perform valve isolation trace
+
+Run a filtered trace to locate operable features that will isolate an area from the flow of network resources.
+
+![Image of a utility network with an isolation trace applied to it](PerformValveIsolationTrace.png)
```

Comment:
I guess this alt text is already approved from the design, but it's different to how we did our other samples which were of the standard "Image of" + name of the sample. Just an observation, I like this description since it's more clear.

Reply:
Yep, I've made this uniform with the current standard.